store/memory_test.go

```diff
@@ -100,3 +102,56 @@ func TestExpiration(t *testing.T) {
 	require.Error(t, err)
 
 }
+
+func TestLatency(t *testing.T) {
+	t.Parallel()
+
+	putLatency := 1 * time.Second
+	getLatency := 1 * time.Second
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	kzgConfig := &kzg.KzgConfig{
+		G1Path:          "../resources/g1.point",
+		G2PowerOf2Path:  "../resources/g2.point.powerOf2",
+		CacheDir:        "../resources/SRSTables",
+		SRSOrder:        3000,
+		SRSNumberToLoad: 3000,
+		NumWorker:       uint64(runtime.GOMAXPROCS(0)),
+	}
```

Comment:
knit - can reduce code redundancy across tests by having this struct declared and returned within a single helper function

Reply:
[9000801](https://github.com/Layr-Labs/eigenda-proxy/pull/114/commits/90008016a4d452809557b6dd8698c25d83543b1b)